lib/credit_card_form.dart

```diff
@@ -283,6 +283,7 @@ class _CreditCardFormState extends State<CreditCardForm> {
                 textInputAction: TextInputAction.done,
                 onEditingComplete: () {
                   onCreditCardModelChange(creditCardModel);
+                  FocusScope.of(context).unfocus();
```

Comment:
I think it is better to use the node used there like so:
`cardHolderNode.unfocus()` and if that isn't working `FocusScope.of(context).requestFocus(new FocusNode())` this is because I noticed that unfocus() sometimes doesn't work.
This should be important because it is a bug, not a new feature.

Reply:
@VladRoscaDev in my opinion @guicarvalho did the right implementation.
Please refer [this](https://github.com/flutter/flutter/issues/7247#issuecomment-488307127) for more info.